lib/resources/example_json/soarian_examples_diagnostic_report.rb

```diff
@@ -237,6 +226,66 @@ module Resources
                 "performer": {}
             }
         },
+        {
+            "resource": {
+                "resourceType": "DiagnosticReport",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;span ccdRefId=\"ZCCD-Results.Urinalysis.11290\">&lt;b>Color of Urine &lt;/b>Light Yellow &lt;i> (YELLOW )&lt;/i>&lt;/span>"
+                },
+                "identifier": [
+                    {
+                        "use": "official",
+                        "value": "ZCCD-Results.Urinalysis.11290"
+                    }
+                ],
+                "status": "final",
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://loinc.org",
+                            "code": "5778-6"
+                        }
+                    ]
+                },
+                "subject": {
+                    "reference": "Patient/7568F2F3-FC76-4185-A540-1DB56331A387"
+                },
+                "effectiveDateTime": "2017-02-01",
+                "issued": "2017-02-01T13:45:00",
+                "performer": {}
+            }
+        },
+        {
+            "resource": {
+                "resourceType": "DiagnosticReport",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;span ccdRefId=\"ZCCD-Results.Urinalysis.11291\">&lt;b>Color of Urine &lt;/b>Dark Yellow &lt;i> (YELLOW )&lt;/i>&lt;/span>"
+                },
+                "identifier": [
+                    {
+                        "use": "official",
+                        "value": "ZCCD-Results.Urinalysis.11291"
+                    }
+                ],
+                "status": "final",
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://loinc.org",
+                            "code": "5778-6"
+                        }
+                    ]
+                },
+                "subject": {
+                    "reference": "Patient/7568F2F3-FC76-4185-A540-1DB56331A387"
+                },
+                "effectiveDateTime": "2017-02-01",
+                "issued": "2017-02-01T13:30:00",
+                "performer": {}
+            }
+        },
```

Comment:
There are now 16 results in this example section which is a lot of scrolling for a user. Probably 4 or 6 is fine.

Reply:
11 were removed
https://github.com/cerner/fhir.cerner.com/pull/198/commits/d9db9b7e4b7929a5eebdf78431a092938fc7421f#diff-7090a4407d4518986a604ffb34d99bd3L139